bundle/config/resources/grant.go

```diff
@@ -0,0 +1,9 @@
+package resources
+
+// Grant holds the grant level settings for a single principal in Unity Catalog.
+// Multiple of these can be defined on any resource.
+type Grant struct {
```

Comment:
it might be a bit out of scope of this PR, but why it's named Grant and not Permissions like for the rest of resources?

Reply:
Based on our discussion last week we decided grants & permissions will remain separate in DABs!